pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -304,7 +305,7 @@ func (srv AlertmanagerSrv) RoutePostAlertingConfig(c *contextmodel.ReqContext, b
 		return response.Error(http.StatusConflict, err.Error(), err)
 	}
 
-	return ErrResp(http.StatusInternalServerError, err, "")
+	return response.ErrOrFallback(http.StatusInternalServerError, err.Error(), err)
```

Comment:
AFAIK, The error will be included into the message. Let's just add some meaningful fallback message like `Failed to save alertmanager configuraiton`

Reply:
This change actually broke a test surprisingly since we don't covert all error cases in the API error conditionals. So the message changes. I've reverted it back to `err.Error()` for backwards compatibility for now.